.github/workflows/version-and-release.yml

```diff
@@ -0,0 +1,52 @@
+name: Version and Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      newversion:
+        description: 'Semantic Version Bump Type (major minor patch)'
+        required: true
+
+env:
+  node_version: 14
+
+jobs:
+  version_and_release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # fetch full history so things like auto-changelog work properly
+        fetch-depth: 0
+    - name: Use Node.js 14
+      uses: actions/setup-node@v1
+      with:
+        node-version: "14.x"
+        # setting a registry enables the NODE_AUTH_TOKEN env variable where we can set an npm token.  REQUIRED
+        registry-url: 'https://registry.npmjs.org'
+    - run: npm i
+    - run: npm run build --if-present
+    - name: Version and Publish to npm
+      uses: bcomnes/npm-bump@v2.0.2
```

Comment:
I created a fork of this action here: https://github.com/wpilibsuite/npm-bump

We should use the fork for security.

Reply:
sounds good. i'll switch it